src/components/molecules/AdminCheckbox/AdminCheckbox.tsx

```diff
@@ -14,7 +14,8 @@ export interface AdminCheckboxProps
   className?: string;
   errors?: FieldErrors<FieldValues>;
   label?: ReactNode | string;
-  labelPosition?: "before" | "after";
+  labelAfter?: string;
+  labelBefore?: string;
```

Comment:
What happens when both are set to true?
This is not an orthogonal thing where 4 combinations are equally valid, but an enumeration of two values. 

```suggestion  
  labelPosition?: "before" | "after";
```